bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/api2/Shelly2ApiClient.java

```diff
@@ -343,6 +346,9 @@ private boolean updateBreakerStatus(int id, ShellySettingsStatus status, @Nullab
         if (bs == null) {
             return false;
         }
+        if (bs.id == null) { // invalid for fw 1.6.1
+            bs.id = id;
+        }
```

Comment:
This should not appear in this PR, please rebase. It was already added by #18797.

Reply:
Did you update your fork, and pull the latest changes from main before rebasing?

Reply:
This is still an issue and needs to be fixed before merging.